common/logger/src/struct_log.rs

```diff
@@ -52,7 +52,7 @@ const CONNECTION_TIMEOUT_MS: u64 = 5000;
 // Fields to keep when over size
 static FIELDS_TO_KEEP: &[&str] = &["error"];
 
-#[derive(Default, Serialize)]
+#[derive(Debug, Default, Serialize)]
```

Comment:
Why implement Debug for StructuredLogEntry?

Reply:
No reason not to, that and I used it to help me debug while writing this patch